portfolio/src/main/webapp/style.css

```diff
@@ -1,34 +1,36 @@
 #name {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 300px;
 }
 
 #image1 {
   display: block;
   text-align: center;
   float: left;
+  position: absolute;
```

Comment:
Add the css rules in alphabetical order. Makes it easier to quickly scan than the properties in a rule. Do the same in the other rules in the file

display:
float:
position:
text-align:

Reply:
Done!